src/platform/tizen/iotjs_module_i2c-tizen.c

```diff
@@ -0,0 +1,131 @@
+/* The MIT License (MIT)
```

Comment:
@LaszloLango `iotjs_module_i2c-tizen.c`does not have any code referenced from this license. This is a license for `iotjs_module_i2c-linux.c`.